packages/nuxt/src/app/composables/asyncData.ts

```diff
@@ -38,7 +38,9 @@ export interface AsyncDataOptions<
   DefaultT = null,
 > {
   server?: boolean
+  /** @deprecated Use strategy: 'lazy' */
```

Comment:
Maybe we could deprecate the composable too ? 
`useLazyFetch`, `useLazyAsyncData`